Documentation/commands.md

```diff
@@ -253,7 +253,17 @@ sudo ./rkt run --volume work,kind=host,source=/opt/tenant1/work \
 
 #### Customize Networking
 
-Work in progress. Please contribute!
+The default networking configuration for Rocket is "host networking". This means that the apps within the pod will share the network stack and the interfaces with the host machine. Since the metadata service uses the container IP for identity, rkt will not register the container with the metadata service in this mode.
+
+Another common configuration, "private networking", means the pod will be executed with its own network stack. This is similar to the how other container tools work. For all of the private networking options the metadata service, launched via `rkt metadata-service`, must be running.
```

Comment:
"For all of the private networking options the metadata service, launched via `rkt metadata-service`, must be running." -- this sentence is unclear.